keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1104,11 +1086,6 @@ repeated evaluations if they are shared across validations.
 
 #### Secondary Authz
 
-<<[UNRESOLVED jpbetz, deads2k, tallclair, ?? ]>>
-We have general agreement to include this as a feature, but need to provide a
```

Comment:
Is just deleting this right? Include a hint that this isn't final yet?

Reply:
Thx, added.